index.html

```diff
@@ -49,5 +49,167 @@ <h2>I'am a Sotfware Developer</h2>
           </div>
         </section>
     </main>
+    <div class="main-body">  
+      <section class="main-row">
+        <h1>My Recent work</h1>
+        <div class="h-line"><hr></div>
+        <div class="story-section">
+          <img src="images/st-img.png" alt="st-imag">
+          <div class="h1-tag"><h1>Multi-Post Stories</h1></div>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        </div>
+        <div class="lang-btn">
+          <button type="button">css</button>
+          <button type="button">html</button>
+          <button type="button">bootstrap</button>
+          <button type="button">Ruby</button>
+        </div>
+        <div class="see-prjct-btn">
+          <button type="button">See Project</button>
+        </div>
+        <div class="pro-art-paint">
```

Comment:
[OPTIONAL] For a better semantic and readable document, you could use the tag `article` instead of the `div` tag to wrap each of your cards in the works section. [You can read more about semantic HTML5 here.](https://www.freecodecamp.org/news/semantic-html5-elements/)